src/components/TrackTile/TrackerDetails/unit-display.ts

```diff
@@ -15,22 +15,28 @@ export const unitDisplay = ({
   tracker,
   skipInterpolation = false,
 }: UnitDisplayConfig) => {
-  // This should never happen. If it does it means we are missing translations.
-  const defaultDisplay = `{{count}} ${unit.display.replace(/^(.)/, (match) =>
+  const defaultOne = `{{count}} ${unit.unit.replace(/^(.)/, (match) =>
```

Comment:
I realize you didn't write this but we could use upperFirst from lodash here which might make it more readable.